Source/Classes/Categories/PINImage+DecodedImage.m

```diff
@@ -224,6 +277,47 @@ UIImageOrientation pin_UIImageOrientationFromImageSource(CGImageSourceRef imageS
     return orientation;
 }
 
+void pin_degreesFromOrientation(UIImageOrientation orientation, void (^completion)(CGFloat degrees, BOOL horizontalFlip, BOOL verticalFlip)) {
```

Comment:
Please use `NS_INLINE` and move it to before it is used near the top of the file.

Reply:
Certainly!